Pipfile

```diff
@@ -6,7 +6,20 @@ name = "pypi"
 [packages]
 sgqlc = {editable = true,path = "."}
 graphql-core = "*"
-websocket-client = "==0.56.0"
+flake8 = "*"
+pep8 = "*"
+flake8-builtins = "*"
+flake8-colors = "*"
+flake8-comprehensions = "*"
+flake8-deprecated = "*"
+flake8-rst-docstrings = "*"
+flake8-requirements = "*"
+flake8-single-quotes = "*"
+flake8-regex = "*"
+flake8-module-name = "*"
+flake8-tidy-imports = "*"
+flake8-print = "*"
```

Comment:
ouch, these are dev dependencies. Please undo thses changes and use `pipenv install -d` (-d for development)

Reply:
Updated the pipfile and squashed commit messages.  Please re review.